docs/coach.rst

```diff
@@ -1,6 +1,8 @@
-Coach
-=====
+.. _coach:
 
+Coach Learners in Kolibri
```

Comment:
A part from the inconsistency you already noted, it is generally recommended to avoid verbs in continuous (gerund) forms if possible, because they can make translators job more difficult. I was also considering the solution `Coach your Learners with Kolibri`, does that sound better?

Reply:
Changed to @radinamatic 's suggestion, so we don't risk the translation confusion for now.

I would like to question the mentioning of "Kolibri" in every section heading... "Install Kolibri", "Access Kolibri", "Manage Kolibri", "Learn With Kolibri" etc -- it just becomes a bit too repetitive in a coherent menu. If you agree, we can fix this..... later :)